test/Field.test.tsx

```diff
@@ -1,545 +1,352 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import { Formik, Field, FieldProps, FormikProps } from '../src';
+import { cleanup, render, wait } from 'react-testing-library';
+import {
+  Formik,
+  Field,
+  FastField,
+  FieldProps,
+  FieldConfig,
+  FormikProps,
+  FormikConfig,
+  FastFieldProps,
+} from '../src';
 
-import { shallow, mount } from 'enzyme';
 import { noop } from './testHelpers';
 
-interface TestFormValues {
-  name: string;
-  email: string;
+const initialValues = { name: 'jared', email: 'hello@reason.nyc' };
+type Values = typeof initialValues;
+
+function renderForm(ui: any, props?: Partial<FormikConfig<Values>>) {
```

Comment:
```suggestion
function renderForm(ui: any, props?: Partial<FormikConfig<Values>>) {
function renderForm(ui: React.ReactNode, props?: Partial<FormikConfig<Values>>) {
```

Reply:
I also think we should maybe make both optional and reverse the order

Reply:
We should also share these between Field and Formik in a perfect world

Reply:
I wanted to keep it as close as possible to original `render` from `react-testing-library`, e.g. ui as a first argument.